src/decoder.js

```diff
@@ -31,11 +39,13 @@ export const decodeInputData = (inputData) => {
  *
  */
 
-export const decodeLogData = (logData) => {
+export const decodeLogData = (logData, address) => {
```

Comment:
Log data is array of logs, it doesn't have specific address, unless if you mean the transaction address and this will be against the function goal.